lib/helpers.js

```diff
@@ -111,26 +111,31 @@ export const createProfile = async ({
   const {id: profileId} = await profileManager.createProfile(profileOptions);
 
   // update profile and profile agent user with content
-  const {accessManager, profile, profileAgent} = await profileManager
-    .getAccessManager({profileId});
+  const {
+    accessManager, profile, profileAgent
+  } = await profileManager.getAccessManager({profileId});
   const [user] = await Promise.all([
     profileAgentContent && _updateProfileAgentUser(
       {accessManager, profileAgent, profileAgentContent}),
     profileContent && _updateProfileUser(
       {accessManager, profileId, profileContent})
   ]);
 
+  let updatedProfile;
```

Comment:
If we don't do this, then the profile returned at the end could be undefined.

```suggestion
  let updatedProfile = profile;
```